protos/PublicDefs.proto

```
@@ -166,8 +166,13 @@ message TaskToCtld {
 
   string reservation = 36;
   bool exclusive = 37;
+<<<<<<< HEAD
```

Buf (1.55.1)

syntax error: unexpected '<'

(COMPILE)